src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/features/SchedulerConfigTest.java

```diff
@@ -18,6 +18,8 @@
     properties = {
         "scheduling.task.delete-dispatched-files.enabled=true",
         "scheduling.task.delete-dispatched-files.cron: */1 * * * * *",
+        "scheduling.task.delete-rejected-files.enabled=true",
```

Comment:
it is enabled by default. perhaps missing the entry in `application.properties` of integration suite?

Reply:
fixed